edx-platform/wikilearn/lms/templates/header/navbar-not-authenticated.html

```diff
@@ -25,13 +25,7 @@
 <nav class="nav-links" aria-label=${_("Supplemental Links")}>
   <div class="main">
   <div class="mobile-nav-item hidden-mobile nav-item nav-tab">
-    <a href="#">${_("Catalog")}</a>
-  </div>
-  <div class="mobile-nav-item hidden-mobile nav-item nav-tab">
-    <a href="#">${_("About")}</a>
-  </div>
-  <div class="mobile-nav-item hidden-mobile nav-item nav-tab">
-    <a href="#">${_("Contact Us")}</a>
```

Comment:
i think instead of deleting just comment out these because we need them in the near future.